Hogajama/configuration/standalone-openshift.xml

```diff
@@ -140,6 +140,14 @@
                         <password>sa</password>
                     </security>
                 </datasource>
+                <datasource jndi-name="java:jboss/datasources/Hogajama" pool-name="HogajamaDS" enabled="true">
+                    <connection-url>jdbc:postgresql://postgresql:5432/${env.POSTGRESQL_DATABASE}</connection-url>
+                    <driver>postgresql</driver>
+                    <security>
+                        <user-name>${env.POSTGRESQL_USER}</user-name>

```

Comment:
POSTGRESQL_USER, POSTGRESQL_DATABASE and POSTGRESQL_PASSWORD are uses but nowhere defined? How can this work? 
I don't see a reason to use variables for POSTGRESQL_USER and POSTGRESQL_DATABASE. Since this database is used only for this service, we don't need configuration?

Reply:
The user **must** be hogajama, otherwise Flyway migration won't work.